extension/app/ts/background/background.ts

```diff
@@ -539,7 +539,7 @@ async function onContentScriptConnected(port: browser.runtime.Port) {
 	})
 }
 
-type PopupMessageHandler = (simulator: Simulator, request: PopupMessage) => Promise<void>
+type PopupMessageHandler = (simulator: Simulator, request: PopupMessage) => Promise<unknown>
```

Comment:
I'm not a huge fan of having some functions return a value while others don't.  I also don't see the return value being used anywhere in this PR, but maybe I'm just missing it?  If it isn't used, then this should stay as `Promise<void>` I think.

Reply:
Its not used anywhere. That's why I don't see the point in requiring it to be void. The return value doesn't matter

Reply:
You can assign a function that returns a value to a function that returns void, so I think `Promise<void>` is correct here?

Reply:
Oh, this is a variance issue with generics.  I remember this now... 😖

Reply:
That's other way to do it, but it requires more boilerplate

Reply:
No, your way is right I think.  One could create a new return type like `type Awaitable = Promise<unknown>` and then use that as the acceptable return type, but I'm not sure that is any better.